feat(changelog): secret-manager-changed-secret-manager-console-pages-hav 2023-11-23 (#2443)
scaleway/docs-content: changelog/november2023/2023-11-23-secret-manager-changed-secret-manager-console-pages-hav.mdx

---
title: Secret Manager console pages have been updated
status: changed
author:
    fullname: 'Join the #secret-manager channel on Slack.'

>     url: 'https://slack.scaleway.com/T7YEXCR7X/C04KGMME3U1'

Reviewer: ```suggestion
    url: 'https://slack.scaleway.com'
```

date: 2023-11-23
category: security-identity
product: secret-manager
---

> In the Secret Manager section of the console, you can now select a region to display secrets and folders inside this region. Whenever a secret is created, there is no more need to specify the secret, as it will be created in the region you are currently working on.

Reviewer: ```suggestion
In the Secret Manager section of the [Scaleway console](https://console.scaleway.com), you can now select a [region](/identity-and-access-management/secret-manager/concepts/#region) to display secrets and folders inside this region. Whenever you create a secret, you no longer need to specify the region, as it will be created in the region you are currently working on.
```

> Read more about Secret Manager logs in our [dedicated documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/concepts/)

Reviewer: ```suggestion
Find out more about Secret Manager in our [dedicated documentation](/identity-and-access-management/secret-manager/quickstart/).
```

> Read more about Secret Manager logs in our <Link to=https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/concepts/>documentation</Link>

Reviewer 2: ```suggestion
Read more about Secret Manager logs in our [dedicated documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/concepts/)
```

> Read more about Secret Manager logs in our <Link to=https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/concepts/>documentation</Link>

Reviewer: ```suggestion
Find out more about Secret Manager in our [dedicated documentation](/identity-and-access-management/secret-manager/quickstart/).
```
